services/__tests__/storage.test.ts

```diff
@@ -0,0 +1,77 @@
+import { BlobItem } from '@azure/storage-blob';
+import { getLastBlob, listBlobs } from '../storage';
+
+const BLOB_NAME = "blob.json"
+
+it('It should sucessfully featch a blob', async () => {
+
+  const expectedBlob = {
+    output: {
+      result: "sucess",
+      messages: [
+        {
+          code: "200",
+          message: "ok",
+          severity: "info"
+        }
+      ],
+      body: {
+        aliases: [
+          {
+            id: 0,
+            participantID: "string",
+            country: "string",
+            alias: "string",
+            language: "string"
+          }
+        ]
+      }
+    }
+  }
+  const containerClientStub = {
+    listBlobsFlat: async function* (): AsyncIterableIterator<BlobItem> {
+      yield expectedBlob as unknown as BlobItem;
+    }
+  };
+
+  const blobClientStub = {
+    getContainerClient: (_: any) => containerClientStub
+  };
+
+  const res = await listBlobs(blobClientStub as any, BLOB_NAME).next();
+
+  expect(res.value).toBe(expectedBlob)
+});
+
+it('It should sucessfully featch a blob', async () => {
```

Comment:
can you differentiate the description from the previous test?

Reply:
My fault, I'll add a correct description 👍